src/graphql/user/Resolvers.js

```diff
@@ -0,0 +1,73 @@
+const LOG = require('../../utils/Log');
+const connection = require('../../db');
+
+const Resolvers = {
+    Query: {
+        async getConfig(root, params) {
+            const query = {
+                text: "SELECT configuration FROM user_config WHERE username=$1 AND tenant=$2;",
+                values: [params.user, params.tenant]
+            };
+
+            try {
+                const result = await connection.query(query);
+                if (result.rowCount) {
+                    return (JSON.stringify(result.rows[0].configuration));
```

Comment:
If I return a row, but the configuration value is null?

Reply:
I assumed it wouldn't happen since in the table configs this field was defined as not null, but it is something to pay attention to in the insert part of this code .